src/util/ringct.rs

```diff
@@ -264,6 +288,23 @@ pub struct RctSigBase {
     pub out_pk: Vec<CtKey>,
 }
 
+impl Display for RctSigBase {
+    fn fmt(&self, fmt: &mut Formatter<'_>) -> Result<(), FmtError> {
+        writeln!(fmt, "RCT_type: {}", self.rct_type)?;
```

Comment:
This structure still have `_`, the rest looks good

Reply:
Done